mk/spksrc.service.installer.dsm7

```diff
@@ -60,7 +69,7 @@ fi
 
 
 # Load (wizard) variables stored by postinst
-call_func "load_variables_from_file" ${INST_VARIABLES}
+call_func "load_variables_from_file" install_log ${INST_VARIABLES}
```

Comment:
how do we call `load_variable_from_file` in a service-setup.sh file of a package?
I might be called in `service_prestart` and must not log to the installer log file.

Reply:
I don't think we do directly. I only see calls in the `service.installer.dsm` files. A search of the repository only shows references to `load_variables_from_file` in these files.

Reply:
This is used in minio package
the function is currently called `export_variables_from_file` but might be replaced by `load_variables_from_file`.

Reply:
And it was my intention to use a parameter for the file, to use is with other files that `${INST_VARIABLES}` as minio does.

Reply:
but this is not an issue, as `install_log` is not a parameter of `load_variables_from_file` but a parameter of `call_func`.
Sorry for the misleading...

Reply:
and I can confirm that no `service-setup.sh` uses `call_func`.